### DISCUSSION TITLE
interactive mode gpu utilization

### DISCUSSION THREAD

**Discussion by d7zda** [ggml-org/llama.cpp#4144] [Q&A]
is it normal for the gpu% shown under rocm-smi to remain 100% during interactive mode while idling? (not sending prompts or outputs)
i feel like the usage should be closer to 0% while the vram% should still indicate the loaded LLM mem usage.

i recently setup stable-diffusion-webui with the deforum extension and it seemed to jack up rocm a bit, because even after i close the terminal sessions and log out then log back in, it still shows 100% gpu usage with the high avgpwr wattage.

im concered about the high avgpwr, i dont want high power consumption or gpu usage with little activity

nov 24 edit: 
i remember a couple of days ago of chatting with the model and the PC was very quiet, the fans were silent and the gpu AVGPWR did not budge from 110w more or less. 

Now, its everytime that i prompt the fans increase their speed like crazy and the gpu avgpwr shoots up to near cap of around 320w or so, then when it completes the response it settles back down.

is there a setting to get it like before?

**Comment by KerfuffleV2** [comment]
Looks like this may be related to #3929.
